src/main/scala/com/campudus/tableaux/controller/MediaController.scala

```diff
@@ -273,14 +283,18 @@ class MediaController(override val config: TableauxConfig,
 
   def mergeFile(uuid: UUID, langtag: String, mergeWith: UUID): Future[ExtendedFile] = {
     for {
+    // retrieve cells with this file for cache invalidation
+      cellsForFile1 <- attachmentModel.retrieveCells(uuid)
+      cellsForFile2 <- attachmentModel.retrieveCells(uuid)
```

Comment:
Same question as above - first retrieve / delete and then do everything to invalidate cache?